src/sidebar/store/modules/selection.js

```diff
@@ -8,6 +8,12 @@
  * - The current filter query
  */
 
+/**
+ * @typedef User
+ * @property {string} username - Unique user's id
```

Comment:
Is this a `userid` or a `username`? Need to be careful with terminology. Otherwise, 👍 on `typedef` here.

Reply:
this sorta drives me nuts, its originally called a `userid` from lms app, then morphs into a `username` at the point of the rpc call.

```
{"grading": {"assignmentName": "Hypothesis LMS", "courseName": "Test Course", "students": [{"LISOutcomeServiceUrl": "https://blackboard.hypothes.is/webapps/gradebook/lti11grade", "LISResultSourcedId": "bbgc34gi59", "displayName": "kyle_student3 K3 Student", "userid": "acct:45a3f1713ca9df0e52d0c9651f115c@lms.hypothes.is"}, {"LISOutcomeServiceUrl": "https://blackboard.hypothes.is/webapps/gradebook/lti11grade", "LISResultSourcedId": "bbgc33gi59", "displayName": "kyle_student2 K2 Student", "userid": "acct:df872ba3bd3fe8a978e75612e89321@lms.hypothes.is"}, {"LISOutcomeServiceUrl": "https://blackboard.hypothes.is/webapps/gradebook/lti11grade", "LISResultSourcedId": "bbgc28gi59", "displayName": "kyle_student Student Test", "userid": "acct:7c7235a6ae4305194be71922c543b3@lms.hypothes.is"}]}, "lmsGrader": true,
... }}
```

Reply:
```
rpcCall(sidebar.frame, sidebar.origin, 'changeFocusModeUser', [
          {
            username: user.userid,
            displayName: user.displayName,
          },
        ]);
```

Reply:
What's the origin of the code that you pasted:

```
rpcCall(sidebar.frame, sidebar.origin, 'changeFocusModeUser', [
          {
            username: user.userid,
            displayName: user.displayName,
          },
        ]);
```

It should be `userid` there, not `username`

Reply:
This is from the LMS app itsself, callback when you change students. I'm thinking this should be userid